saleor/graphql/checkout/mutations/checkout_add_promo_code.py

```diff
@@ -120,6 +121,12 @@ def perform_mutation(  # type: ignore[override]
             recalculate_discount=False,
             save=True,
         )
-        cls.call_event(manager.checkout_updated, checkout)
+        call_checkout_event_for_checkout_info(
+            manager=manager,
+            event_func=manager.checkout_updated,
```

Comment:
I am aware this is "ugly" way of passing arguments. Instead of passing `=manager.checkout_updated,` and `event_name=WebhookEventAsyncType.CHECKOUT_UPDATED,` the best way would be passing single variable. I will address this in a separate PR. First, need to figure out a better way of doing this.

Reply:
Added this to my to-do list for the task: https://linear.app/saleor/issue/MERX-156/application-with-order-updated-webhook-subscribe-for-ordertotal-may